_tests/tests_suite_test.go

```diff
@@ -27,31 +27,55 @@ func init() {
 }
 
 func getRandAppName() string {
-	return fmt.Sprintf("apps-test-%d", rand.Int())
+	name := fmt.Sprintf("test-%d", rand.Int())
+	if len(name) > 24 {
+		name = name[:24]
+	}
+	return name
 }
 
 func TestTests(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Tests Suite")
+	RunSpecs(t, "Deis Workflow")
 }
 
 var (
-	testAdminUser     = fmt.Sprintf("test-admin-%d", GinkgoConfig.RandomSeed)
-	testAdminPassword = "asdf1234"
-	testAdminEmail    = fmt.Sprintf("test-admin-%d@deis.io", GinkgoConfig.RandomSeed)
+	testAdminUser     = "admin"
+	testAdminPassword = "admin"
+	testAdminEmail    = "admin@example.com"
```

Comment:
Do tests fail spectacularly (fwiw, a good thing IMO) if this user already exists (e.g. the last test didn't clean up properly)?